.github/workflows/wp-coding-standards.yml

```diff
@@ -0,0 +1,44 @@
+name: WordPress Coding Standards
```

Comment:
The previous action needed to use a GH Token, are these new scripts token-free? If so, that's really cool.

Reply:
As written here, the actions work :)